src/site/content/en/learn/css/backgrounds/index.md

```diff
@@ -0,0 +1,456 @@
+---
+title: Backgrounds
+description: >
+  In this module learn the ways you can style backgrounds of CSS boxes using CSS
+audio:
+  title: The CSS Podcast - 053: Background
+  src: https://traffic.libsyn.com/secure/thecsspodcast/TCP052_v1.mp3?dest-id=1891556
+  thumbnail: image/foR0vJZKULb5AGJExlazy1xYDgI2/ECDb0qa4TB7yUsHwBic8.png
+authors:
+ - lozandier
+date: 2021-9-15
```

Comment:
This probably works, but can you write 2021-09-15 just in case?

Reply:
```suggestion
date: 2021-11-09
```